README.md

```diff
@@ -173,6 +173,12 @@ This project is build with:
 - Twitter: [@Evans_22J](https://twitter.com/Evans_22J)
 - LinkedIn: [@Evans Sitibekiso](https://www.linkedin.com/in/evans-sitibekiso/)
 
```

Comment:
- [ ] It would be better if we write the redirect test for our User Show page as shown in the screenshot below

![Screenshot from 2022-08-04 17-00-27](https://user-images.githubusercontent.com/46388113/182908693-59ed2b8b-c1b2-4542-b9d3-b932b50ba9f9.png)